examples/hooks-next/package.json

```diff
@@ -22,6 +21,6 @@
     "@types/react": "^17.0.37",
     "eslint": "8.4.0",
     "eslint-config-next": "12.0.7",
-    "typescript": "^4.5.2"
+    "typescript": "4.5.4"
```

Comment:
will we have some way to keep those in sync? or just paying attention when adding/updating an example to deduplicate?

Reply:
It's not a big deal if we have different TypesScript versions here. I took this opportunity to bump it to the latest and to pin it.